website/docs/common-scripts/00-common-scripts-introduction.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: introduction
+title: Introduction
+---
+
+# Introduction
+
```

Comment:
There should be a short description on what this section is and why we collect these scripts. The danger Disclaimer should be moved to the end of the brief description here, too.